src/commands/debug_files/create_jvm_based_bundle.rs

```diff
@@ -0,0 +1,110 @@
+use std::fs;
+use std::path::{Path};
+use anyhow::{bail, Context, Result};
+use clap::{Arg, ArgMatches, Command};
+use symbolic::debuginfo::sourcebundle::{SourceFileType};
+use crate::api::Api;
+use crate::config::Config;
+use crate::utils::args::ArgExt;
+use crate::utils::file_search::ReleaseFileSearch;
+use crate::utils::file_upload::{FileUpload, SourceFile, UploadContext};
+use crate::utils::fs::{path_as_url};
+
+pub fn make_command(command: Command) -> Command {
+    command
+        .hide(true) // experimental for now
+        .about("Create a source bundle for the given JVM based source files (e.g. Java, Kotlin, ...)")
+        .org_arg()
+        .project_arg(false)
+        .arg(
+            Arg::new("path")
+                .value_name("PATH")
+                .required(true)
+                .help("The directory containing source files to bundle."),
+        )
+        .arg(
+            Arg::new("output")
+                // .short('o')
+                .long("output")
+                .value_name("PATH")
+                .required(true)
+                .help("The path to the output folder."),
+        )
+        .arg(
+            Arg::new("debug_id")
+                // .short('d')
+                .long("debug-id")
+                .value_name("UUID")
+                .required(true)
+                .help("Debug ID to use for the source bundle."),
+        )
+}
+
+pub fn execute(matches: &ArgMatches) -> Result<()> {
+    let config = Config::current();
+    let org = config.get_org(matches)?;
+    let project = config.get_project(matches).ok();
+    let api = Api::current();
+    let chunk_upload_options = api.get_chunk_upload_options(&org)?;
+    let context = &UploadContext {
+        org: &org,
+        project: project.as_deref(),
+        release: None,
+        dist: None,
+        note: None,
+        wait: true,
+        dedupe: false,
+        chunk_upload_options: chunk_upload_options.as_ref(),
+    };
+    let path = Path::new(matches.get_one::<String>("path").unwrap());
+    let output_path = Path::new(matches.get_one::<String>("output").unwrap());
+    let debug_id = matches.get_one::<String>("debug_id").unwrap();
```

Comment:
```suggestion
    let debug_id = matches.get_one::<Uuid>("debug_id").unwrap();
```

+ update the flag definition to include `value_parser(Uuid::parse_str)`, then you can remove whole parsing below.

Reply:
Could this not be `let debug_id = matches.get_one::<DebugId>("debug_id").unwrap();` directly, using `DebugId::parse_str`?

Reply:
Yup, good catch. Although it'll be `.value_parser(DebugId::from_str)`